packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.module.scss

```diff
@@ -12,3 +13,63 @@
 .icon {
   margin-left: $spacing-xs;
 }
+
+.reversed {
+  border: $border-solid-border-width $border-solid-border-style
+    rgba($color-white-rgb, 0.8);
+  background-color: transparent;
+  color: $color-white;
+
+  &.placeholder {
+    color: rgba($color-white-rgb, 0.65);
+  }
+
+  .icon {
+    color: rgba(255, 255, 255, 0.8);
+  }
+
+  #{$story-className--button-active},
+  #{$selectors--button-hover} {
+    background-color: rgba($color-white-rgb, 0.1);
+    border: $border-solid-border-width $border-solid-border-style $color-white;
+
+    .icon {
+      color: $color-white;
+    }
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  #{$selectors--button-focus} {
+    background-color: rgba($color-white-rgb, 0.1);
```

Comment:
The [UI kit](https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%EF%B8%8F%F0%9F%96%BC%EF%B8%8F-Component-Gallery?node-id=1206%3A155907&t=rVqn5LiNNw3fUPxn-0) has 0.2 opacity 

`background-color: rgba($color-white-rgb, 0.2);`